### PR TITLE
EWPP-1367: Add related departments field to news.

### PR DESCRIPTION
**Pull request by nagyad**

### CODE REVIEW

File: modules/oe_theme_content_news/oe_theme_content_news.module [hunk 1]
```diff
@@ -57,6 +57,7 @@ function oe_theme_content_news_node_view_alter(array &$build, EntityInterface $e
     // Rename field labels based on number of items.
     $plural_label_overrides = [
       'oe_author' => t('Authors'),
+      'oe_departments' => t('Related departments'),
```
Comment (22Alexandra) [inline modules/oe_theme_content_news/oe_theme_content_news.module]: In the [requirements page](https://webgate.ec.europa.eu/fpfis/wikis/pages/viewpage.action?pageId=572200293), the label is "Departments".

File: templates/field/field--node--oe-departments--oe-news--full.html.twig [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{#
+/**
+ * @file
+ * Theme override for the Responsible department field template of the News content type.
```
Comment (22Alexandra) [inline templates/field/field--node--oe-departments--oe-news--full.html.twig]:
```suggestion
 * Theme override for the Related department field template of the News content type.
```

File: modules/oe_theme_content_news/config/install/core.entity_view_display.node.oe_news.full.yml [hunk 3]
```diff
@@ -19,7 +20,6 @@ dependencies:
     - field.field.node.oe_news.oe_teaser
     - node.type.oe_news
   module:
-    - datetime
```
Comment (22Alexandra) [inline modules/oe_theme_content_news/config/install/core.entity_view_display.node.oe_news.full.yml:22]: Why?

**Review by 22Alexandra — CHANGES_REQUESTED** [review]

File: modules/oe_theme_content_news/config/install/core.entity_view_display.node.oe_news.full.yml (hunk 3 above)
Comment (nagyad) [reply modules/oe_theme_content_news/config/install/core.entity_view_display.node.oe_news.full.yml:22]: It was automatic.

File: modules/oe_theme_content_news/oe_theme_content_news.module (hunk 1 above)
Comment (nagyad) [reply modules/oe_theme_content_news/oe_theme_content_news.module]: Yeah but as I explained in a comment of the ticket I followed our pattern of handling prular and singular labels

Comment (22Alexandra) [reply modules/oe_theme_content_news/oe_theme_content_news.module]: My remark is about the label itself that should be "Department" for singular and "Departments" for plural, not the implementation.

**Review by 22Alexandra — DISMISSED** [review]

**Review by sergepavle — DISMISSED** [review]

**Review by 22Alexandra — APPROVED** [review]

**Review by sergepavle — APPROVED** [review]

**Review by sergepavle — APPROVED** [review]

**Pull request merged**